docs/materials/osg/ex3-submit-osg.md

```diff
@@ -25,6 +25,7 @@ Now to create submit files that will run in the OSG!
         connect project
 
     If you are given a choice and don't know which project to select, reach out to OSG staff.
+    The default project may be changed at a later time, or overriden in your submit files.
```

Comment:
What do you think of adding just a few more words? “The default project will stay in effect until changed at a later time.”

Also, the second part raises the obvious question of “how?”. So maybe it is best to address that question (if simple) or omit the tease.